### ISSUE TITLE
Update instructions

### ISSUE THREAD

**Issue by jwillmer** [copercini/esp8266-aws_iot#6]
I find it hard to follow your instructions. I used the example `MQTT_x509_bin.ino` and found out that `xxd` is a Linux command that transforms the converted DER file to byte code. I downloaded `xxd` for Windows and did this. Now I tested the code and i get a `-4 ` error back from AWS. I also found the [AWSconnectivity diagnose website](https://docs.aws.amazon.com/iot/latest/developerguide/diagnosing-connectivity-issues.html) but there I need `PEM` files. 

Could you update the documentation to make it easier to follow? Maybe I can then figure out what I did wrong in order to get a `-4` error. Do you know there I can look the error codes up?

**Comment by chegewara** [comment]
Thanks, you are life saver.
